doc/releases/changelog-0.11.0.md

```diff
@@ -23,175 +27,235 @@
 
-      return qml.expval(qml.Z(0))
-  ```
+        qml.cond(x > 1.4, ansatz_true, ansatz_false)()
 
-* Catalyst now supports experimental capture of PennyLane transforms.
-  [(#1544)](https://github.com/PennyLaneAI/catalyst/pull/1544)
-  [(#1561)](https://github.com/PennyLaneAI/catalyst/pull/1561)
-  [(#1567)](https://github.com/PennyLaneAI/catalyst/pull/1567)
-  [(#1578)](https://github.com/PennyLaneAI/catalyst/pull/1578)
+        return qml.expval(qml.Z(0))
+    ```
 
-  To trigger the PennyLane pipeline for capturing the mentioned transforms,
-  simply set `experimental_capture=True` in the qjit decorator. If available,
-  Catalyst will apply its own pass in replacement of the original transform
-  provided by PennyLane. Otherwise, the transform will be expanded according
-  to PennyLane rules.
+    ```pycon
+    >>> circuit(0.1)
+    Array(0.99500417, dtype=float64)
+    ```
 
-  ```python
-  import pennylane as qml
-  from catalyst import qjit
+  * Support for `qml.for_loop`:
+
+    ```python
+    dev = qml.device("lightning.qubit", wires=2)
+
+    @qjit(experimental_capture=True)
+    @qml.qnode(dev)
+    def circuit(x: float):
+
+        @qml.for_loop(10)
+        def loop(i):
+            qml.H(wires=1)
+            qml.RX(x, wires=0)
+            qml.CNOT(wires=[0, 1])
+
+        loop()
+        return qml.expval(qml.Z(0))
+    ```
+
+    ```pycon
+    >>> circuit(0.1)
+    Array(0.97986841, dtype=float64)
+    ```
+
+  * Support for `qml.while_loop`:
+
+    ```python
+    @qjit(experimental_capture=True)
+    @qml.qnode(dev)
+    def circuit(x: float):
+
+        f = lambda c: c < 5
+
+        @qml.while_loop(f)
+        def loop(c):
+            qml.H(wires=1)
+            qml.RX(x, wires=0)
+            qml.CNOT(wires=[0, 1])
+
+            return c + 1
+        
+        loop(0)
+        return qml.expval(qml.Z(0))
+    ```
 
+    ```pycon
+    >>> circuit(0.1)
+    Array(0.97526892, dtype=float64)
+    ```
+
+  Additionally, Catalyst can now apply its own compilation passes when equivalent transforms are provided 
+  by PennyLane (e.g., `cancel_inverses` and `merge_rotations`). In cases where Catalyst does not have 
+  its own analogous implementation of a transform available in PennyLane, the transform will be expanded 
+  according to rules provided by PennyLane.
+
+  For example, consider this workflow that contains two PennyLane transforms: `cancel_inverses` and 
+  `single_qubit_fusion`. Catalyst has its own implementation of `cancel_inverses` in the `passes` module, 
+  and will smartly invoke its implementation intead. Conversely, Catalyst does not have its own implementation
+  of `single_qubit_fusion`, and will therefore resort to PennyLane's implementation of the transform.
+
+  ```python
   dev = qml.device("lightning.qubit", wires=1)
 
   @qjit(experimental_capture=True)
-  def func(x: float):
+  def func(r1, r2):
+
+      @qml.transforms.single_qubit_fusion
       @qml.transforms.cancel_inverses
```

Comment:
I think in this order both would be applied on the plxpr, `cancel_inverses` would have to come after `single_qubit_fusion` to be applied in Catalyst.

Reply:
Ah yes you're right :) 

```suggestion
      @qml.transforms.cancel_inverses
      @qml.transforms.single_qubit_fusion
```